recipes/servers.md

```diff
@@ -100,8 +100,10 @@ var apiProxy = proxy('/api', {
 
 module.exports = {
     server: {
+        // Start from key `10` in order to NOT overwrite the default 2 middleware provided
+        // by `lite-server` or any future ones that might be added.
```

Comment:
The list of 2 default middlewares might change or grow in config-defaults.js.

Can you add a link in the comment to the **config-default.js** so devs can see which ones are provided by default. Devs can decide how to mount the proxy middleware to fit their needs.

https://github.com/johnpapa/lite-server/blob/master/lib/config-defaults.js#L16